README.md

```diff
@@ -88,7 +88,7 @@ Simeon Krastnikov,
 Ryan Lee,
 Hayley Lin,
 Bruce Liu,
-Merrick Liu
+Merrick Liu,
```

Comment:
You shouldn't make this change in this PR since it's unrelated to the work you're doing. You would be welcome to make it in a separate PR!